main.py

```diff
@@ -50,18 +52,29 @@ def send_slack_notification(new_value, old_value):
 
 def main():
     latest_value = get_latest_value()
```

Comment:
Vurderte her å lage en tilsvarende fil for SEB, men kan evt legges i en annen commit